GlassFish/5.0-web/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+# Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
+# 
+# GlassFish on Docker with Oracle Linux and OpenJDK
+FROM oraclelinux:latest
```

Comment:
Please do not use `oraclelinux:latest` as this can result in an unexpected build. You should be targeting `oraclelinux:7` or `7-slim`.